OpenRA.Mods.D2k/UtilityCommands/D2kMapImporter.cs

```diff
@@ -84,7 +84,7 @@ public class D2kMapImporter
 			{ 360, Pair.New("light_inf", "Harkonnen") },
 			{ 361, Pair.New("trooper", "Harkonnen") },
 			{ 362, Pair.New("fremen", "Harkonnen") },
-			{ 363, Pair.New("sardaukar", "Harkonnen") },
+			{ 363, Pair.New("mpsardaukar", "Harkonnen") },
```

Comment:
Are you sure this shouldn't stay "sardaukar"? This is mostly for singleplayer maps afterall, so I'd expect most of the initital sardaukar in missions to be of the imperial variety.

Reply:
Some units share entries in original map format and Harkonnen gets MP Sardaukar there, i'm sure. You can't give Harkonnen a preplaced Imperial Sardaukar in original, at least not with D2K+ Map Editor, not sure if there is even hackier way.
![resim](https://user-images.githubusercontent.com/7933210/35780988-c37fcc58-09f4-11e8-984a-dd3a59a0bf22.png)